src/main/java/org/folio/des/scheduling/quartz/converter/bursar/ExportConfigToBursarTriggerConverter.java

```diff
@@ -0,0 +1,119 @@
+package org.folio.des.scheduling.quartz.converter.bursar;
+
+import java.time.LocalDate;
+import java.time.OffsetTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import org.folio.des.domain.dto.ExportConfig;
+import org.folio.des.domain.dto.ScheduleParameters;
+import org.folio.des.scheduling.quartz.QuartzConstants;
+import org.folio.des.scheduling.quartz.converter.ScheduleParametersToTriggerConverter;
+import org.folio.des.scheduling.quartz.trigger.ExportTrigger;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+import lombok.extern.log4j.Log4j2;
+
+@Component
+@Log4j2
+public class ExportConfigToBursarTriggerConverter implements Converter<ExportConfig, ExportTrigger> {
+
+  private final ScheduleParametersToTriggerConverter scheduleParametersToTriggerConverter;
+  private final String timeZone;
+
+  public ExportConfigToBursarTriggerConverter(ScheduleParametersToTriggerConverter scheduleParametersToTriggerConverter,
+                                              @Value("${folio.quartz.bursar.timeZone}") String timeZone) {
+    this.scheduleParametersToTriggerConverter = scheduleParametersToTriggerConverter;
+    this.timeZone = timeZone;
+  }
+
+  @Override
+  public ExportTrigger convert(@NotNull ExportConfig exportConfig) {
+
+    if (isDisabledSchedule(exportConfig)) {
+      return new ExportTrigger(true, Collections.emptySet());
+    }
+    ScheduleParameters scheduleParameters = createBursarScheduleParameters(exportConfig);
+
+    return new ExportTrigger(false, scheduleParametersToTriggerConverter
+      .convert(scheduleParameters, getTriggerGroup(exportConfig)));
+  }
+
+  private boolean isDisabledSchedule(ExportConfig exportConfig) {
+    return Optional.ofNullable(exportConfig.getSchedulePeriod())
+      .map(ExportConfig.SchedulePeriodEnum.NONE::equals)
+      .orElse(true);
+  }
+
+  private ScheduleParameters createBursarScheduleParameters(ExportConfig exportConfig) {
+
+    ExportConfig.SchedulePeriodEnum schedulePeriod = exportConfig.getSchedulePeriod();
+
+    return switch (schedulePeriod) {
+      case HOUR -> buildHourlyScheduleParam(exportConfig);
+      case DAY -> buildDailyScheduleParam(exportConfig);
+      case WEEK -> buildWeeklyScheduleParam(exportConfig);
+      default -> null;
+    };
+  }
+
+  private ScheduleParameters buildWeeklyScheduleParam(ExportConfig exportConfig) {
+    log.info("buildWeeklyScheduleParam::Inside buildWeeklyScheduleParam");
+    ScheduleParameters scheduleParameters = new ScheduleParameters();
+    scheduleParameters.setScheduleFrequency(exportConfig.getScheduleFrequency());
+    scheduleParameters.setTimeZone(timeZone);
```

Comment:
Creating scheduling parameters looks partly identical in 3 methods, could be extracted